next.config.js

```diff
@@ -7,25 +11,42 @@ module.exports = {
   env: {
     API_SERVER: process.env.API_SERVER || 'http://localhost:3999',
   },
+
   webpack(config, { dev, isServer }) {
     const splitChunks = config.optimization && config.optimization.splitChunks;
     if (splitChunks) {
       const cacheGroups = splitChunks.cacheGroups;
-      const reactModules = /[\\/]node_modules[\\/](react|react-dom)[\\/]/;
+      const preactModules = /[\\/]node_modules[\\/](preact|preact-render-to-string|preact-context-provider)[\\/]/;
       if (cacheGroups.framework) {
-        cacheGroups.react = Object.assign({}, cacheGroups.framework, {
-          test: reactModules,
+        cacheGroups.preact = Object.assign({}, cacheGroups.framework, {
```

Comment:
Is there a big gain in switching for preact?

Reply:
![Screen Shot 2020-04-22 at 9 01 47 AM](https://user-images.githubusercontent.com/11803153/79991433-ecf9f400-8477-11ea-9935-faca0096693e.png)

Lighthouse with React:
![Screen Shot 2020-04-22 at 8 58 24 AM](https://user-images.githubusercontent.com/11803153/79991251-ae643980-8477-11ea-8c76-b77268a9a48e.png)

Lighthouse with Preact:
![Screen Shot 2020-04-22 at 9 00 00 AM](https://user-images.githubusercontent.com/11803153/79991273-b9b76500-8477-11ea-8a56-9b7a11a14a34.png)